sample/src/main/res/layout/activity_sample.xml

```diff
@@ -2,6 +2,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:elevation="16dp"
```

Comment:
Setting an elevation on a regular layout isn't something you'd probably do, but you do want to do if for a DrawerLayout. Setting it here means we can see issue #109, and hence see the fix works.